pkg/onit/cli/run.go

```diff
@@ -26,32 +26,39 @@ import (
 
 var (
 	runExample = `
-    # To run all integration tests:
-    onit run suite
+    # To run a single test on the cluster
+    onit run test <name of a test>
 
-    # To run a single test on a cluster
-    onit run test <name of a test>`
+    # To run a suite of tests on the cluster
+    onit run test-suite <name of a suite>
+
+	# To run a benchmark on the cluster
```

Comment:
That would be nice to remove spaces here

Reply:
These aren't spaces. It's tabs vs spaces. Not sure what we should stick with.